src/frontend/src/components/SEO.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+import { useStaticQuery, graphql } from 'gatsby';
+
+function SEO({ description, lang, meta, title }) {
+  const { site } = useStaticQuery(
```

Comment:
Let's use the existing hook for getting this data.  Look at how it's being done in https://github.com/Seneca-CDOT/telescope/blob/78610d4538200146bb5bd51b1cd8281d0f35a06b/src/frontend/src/components/Login/LoggedOut.jsx and you can do the same here.